data/com/mojang/blaze3d/vertex/VertexMultiConsumer.mapping

```diff
@@ -1,8 +1,15 @@
 CLASS com/mojang/blaze3d/vertex/VertexMultiConsumer
+	METHOD create (Lcom/mojang/blaze3d/vertex/VertexConsumer;)Lcom/mojang/blaze3d/vertex/VertexConsumer;
+		ARG 0 consumer
```

Comment:
I'm not sure what this would be since it refers to itself. Maybe `thisInstance`?

Reply:
I would suggest against that, and keep it at `consumer` because `thisInstance` looks to leaks an implementation detail (namely, that the instance is returned if there is only one consumer for delegation).